test/Concurrency/Runtime/async_taskgroup_dontLeakTasks.swift

```diff
@@ -1,8 +1,8 @@
 // RUN: %target-run-simple-swift( -Xfrontend -disable-availability-checking -parse-as-library) | %FileCheck %s
 // TODO: move to target-run-simple-leaks-swift once CI is using at least Xcode 14.3
 
-// This test uses `leaks` which is only available on apple platforms; limit it to macOS:
-// REQUIRES: OS=macosx
```

Comment:
Are you sure we should remove this line?   Does this test really run on Linux and Windows?

Reply:
Yes, this is actually on purpose, but I did miss removing the `import Darwin` 😮‍💨 (that wasn't used, fixed now).

The reason for macos here is explained in the TODO above: `// TODO: move to target-run-simple-leaks-swift once CI is using at least Xcode 14.3` so there's a new way to tests for leaks, but only Xcode 14.3 seems to be reliable enough in reporting them, so once we do that we can make this test use `leaks` and require macos